Atomic/atomic.py

```diff
@@ -808,8 +808,11 @@ def _check_if_image_present(self):
             self.inspect = self._inspect_image()
 
     def install(self):
-        if self._container_exists(self.name):
-            raise ValueError("A container '%s' is already present" % self.name)
+        try:
+            if self._container_exists(self.name):
+                raise ValueError("A container '%s' is already present" % self.name)
+        except NoDockerDaemon:
+            pass
 
```

Comment:
My previous patch set should fix this. Could you verify you are up2date.  

https://github.com/projectatomic/atomic/pull/505